testbed/HIL_Firmware/PDU_HIL/PDU_HIL/userInit.c

```diff
@@ -0,0 +1,178 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "esp_err.h"
+#include "driver/twai.h"
+#include "driver/spi_master.h"
+#include "canReceive.h"
+#include "userInit.h"
+#include "processCAN.h"
+#include "digitalPot.h"
+#include "pduOutputs.h"
+
+spi_device_handle_t pot;
+
+void taskRegister (void)
+{
+    BaseType_t xReturned = pdPASS;
+    TaskHandle_t can_rx_task_handler = NULL;
+    TaskHandle_t can_process_task_handler = NULL;
+    TaskHandle_t relay_pdu_outputs_handler = NULL;
+
+    xReturned = xTaskCreate(
+        can_rx_task,
+        "CAN_RECEIVE_TASK",
+        4000,
+        ( void * ) 1,
```

Comment:
nit: no need to pass in 1

Reply:
Sounds good, I just followed what the freeRtos example had, will change to NULL